tests/test_main.py

```diff
@@ -30,7 +30,7 @@ def test_knowledge_panel_badendpoint():
 
 def test_knowledge_panel_ctegory_with_value_and_country():
```

Comment:
this test needs mocking !
It will never pass more than one day in a raw otherwise !

So you will have to mock the request.get, as you know to do it !

Reply:
Yeah, Just small issue when I run the test it shows a typeerror of "params" so , should I use `@pytest.fixture` or there any other way to do it?